packages/now-python/test/fixtures/14-unicode-handler/probe.js

```diff
@@ -0,0 +1,34 @@
+const fetch = require('../../../../../test/lib/deployment/fetch-retry');
+
+async function fetchDeploymentUrl(url, opts) {
```

Comment:
if we export this function in test/lib/deployment then I don't have to copy it here. lmk.